### DISCUSSION TITLE
[Questions] About Kravatte-WBC

### DISCUSSION THREAD

**Discussion by phantomcraft** [XKCP/XKCP#110] [Q&A] [answered]
What is the maximum key length accepted by Kravate-WBC?

Does increasing the key length turns the encryption slow significantly?

What can be Kravatte-WBC block size? Is it arbitrarily?

**Comment by gvanas** [comment, marked as answer]
The maximum key size of Kravatte and Kravatte-WBC is 320 bits. See Claims 1 and 2 of [our paper](https://tosc.iacr.org/index.php/ToSC/article/view/855) for more details on what it means in terms of claimed security.

The key size has no impact on the performance of Kravatte nor on that of Kravatte-WBC.

The block size of Kravatte-WBC is user-chosen and unlimited.

**Reply by phantomcraft** [reply]
I didn't had enough time to see all the Keccak papers, sorry. Kravate-WBC is very interesting, as I read, it's a truly format-preserving encryption.

Thanks for your answer.
